src/main/java/seedu/address/model/person/Attendance.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Person's attendance in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Attendance {
+    protected static final DateTimeFormatter DTF = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    public final LocalDate time;
+
+    /**
+     * Constructs an {@code Attendance}.
+     *
+     * @param attendance a description of the attendance.
+     */
+    public Attendance(String attendance) {
+        requireNonNull(attendance);
+        this.time = LocalDate.parse(attendance, DTF);
```

Comment:
Would it be better if you added an isValidAttendance method like the other classes? Or something of that sort in the event that LocalDate.parse throws an exception for invalid date format?